### ISSUE TITLE
Add Categories and Tags to all pages in docs

### ISSUE THREAD

**Issue by iArchitSharma** [layer5io/docs#178]
![image](https://github.com/layer5io/docs/assets/74408634/91d6cadd-12c3-4fd0-9aef-e9b2ba15b48f)

Example to add Categories and Tags, simply add it to frontmatter of page:
```
categories: [Designer]
tags: [designs]
```

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by Raj100679** [comment]
@iArchitSharma Can you brief me more, I would like to take this

**Comment by iArchitSharma** [comment]
@Raj100679 yes, please ensure that the Tags and Categories remains relevant to the page

**Comment by Raj100679** [comment]
@iArchitSharma I will work on this will do the necessary changes and raise the pr then we can talk about it
